lib/rack/ecg/check/redis_connection.rb

```diff
@@ -3,17 +3,26 @@ module Rack
   class ECG
     module Check
       # @!method initialize
-      #   Checks whether the global Redis client is currently connected to the
-      #   database.
+      #   Checks whether the given Redis client is currently connected to the
+      #   database as identified by the ++instance++ option.
       #
-      #   Does not take any options.
+      # @option parameters instance [Redis,Hash] Redis parameters to check
```

Comment:
```suggestion
      # @option parameters instance [Redis] The Redis client
```

Passing a hash would fail, so we shouldn't list that as a valid type.